config/jobs/testing/testing-trusted.yaml

```diff
@@ -74,9 +102,11 @@ postsubmits:
       preset-bazel-scratch-dir: "true"
       preset-image-deploy: "true"
       preset-deployer-service-account: "true"
+      preset-deployer-github-token: "true"
+      preset-deployer-ssh-key: "true"
     spec:
       containers:
-      - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20190408-4d1853d-0.24.1
+      - image: eu.gcr.io/jetstack-build-infra-images/bazelbuild:20190409-ac1a0f9-0.24.1
```

Comment:
Image bump is required due to some Bazel build failures. I'm going to 'touch' something in the bazelbuild directory after this merges in order to test out the full script workflow 😄